### ISSUE TITLE
Encouraging survey completion

### ISSUE THREAD

**Issue by crichID** [github/training-kit#424]
**tl;dr** We currently receive end-of-course survey responses from ~18% of the learners who complete the course. We would like to improve this.

## The Problem

In order to improve the course, it would be great to receive more feedback on what learners do and don't like. ~18% is a fairly low response rate - industry norm is ~30%. This could be caused by a few things:

- **Survey Placement:** Right now, the only place the survey appears is at the very bottom of the last page of the course. 
- **Passive Request:** The request will only be seen by people who read every word in the course.

## Potential Solutions

- Should we surface the survey in some other way (or in another place) within the course?
- Should we create a webhook to add a comment to pull requests when they have been merged?
- Other ideas?

**Comment by carolynshin** [comment]
Just an idea...

What if we provided some sort of incentive to fill out the survey?

For example, if we promised some awesome additional learning resources/videos after survey completion. So something like this...
- End of course, link to survey, with a description of the incentive
- Fill out survey
- After survey completion, redirect to cool page of resources that made filling out survey worth it!

**Comment by hectorsector** [comment]
> Should we create a webhook to add a comment to pull requests when they have been merged?

👍 why are all the cool things the ones that require the most work?

**Comment by brianamarie** [comment]
> Should we create a webhook to add a comment to pull requests when they have been merged?

@hectorsector I also have an aspiration to create a webhook to add collaborators after they create an issue, too, since so many people are just creating issues instead of commenting on the correct one. I think this would be a similar script, so I'll put this on my list. (No promises!)



> After survey completion, redirect to cool page of resources that made filling out survey worth it!

👀 Ooooh, @carolynshin, like an easter egg of hidden content? What do you have in mind? I love this idea!

**Comment by VTG38** [comment]
Or Maybe just an @ mention?

**Comment by brianamarie** [comment]
> Or Maybe just an @ mention?

Hm, I'm not sure exactly what is meant by this?

**Comment by crichID** [comment]
> Should we create a webhook to add a comment to pull requests when they have been merged?

@hectorsector perhaps teacher-bot could do this for us?

**Comment by hectorsector** [comment]
This sounds fairly easy to do! [Added](https://github.com/github/teacher-bot/projects/1#card-3024522) to the project after the MVP.

**Comment by crichID** [comment]
@hectorsector now that teacher-bot is configurable, we can easily add this to the comment left by teacher-bot after the PR is merged :tada: I will add a teacherbot.yml file to each of the on-demand repositories to handle this!
